src/Relude/Functor/Reexport.hs

```diff
@@ -15,9 +16,16 @@ module Relude.Functor.Reexport
        , module Data.Functor
        , module Data.Functor.Compose
        , module Data.Functor.Identity
+#if MIN_VERSION_base(4,12,0)
+        , module Data.Functor.Contravariant
+#endif
        ) where
 
 import Data.Bifunctor (Bifunctor (..))
 import Data.Functor (Functor (..), void, ($>), (<$>))
 import Data.Functor.Compose (Compose (..))
 import Data.Functor.Identity (Identity (..))
+
+#if MIN_VERSION_base(4,12,0)
+import Data.Functor.Contravariant
```

Comment:
We stick to the explicit imports policy, could you please specify it?

Reply:
Yes, sorry. I missed that! I'll make the change 👍